### PR TITLE
Create a unified migration docker

### PR DESCRIPTION
**Pull request by ChewingGlass**

### CODE REVIEW

File: Anchor.toml [hunk 1]
```diff
@@ -49,33 +49,23 @@ startup_wait = 20000
 [test.validator]
 url = "https://api.mainnet-beta.solana.com"
 
-[[test.genesis]]
+[[test.validator.clone]]
```
Comment (ThornM9) [inline Anchor.toml:52]: Too many clones slows boot time and sometimes causes timeouts, any reason to stop building from deps?

File: programs/helium-sub-daos/src/instructions/initialize_sub_dao_v0.rs [hunk 2]
```diff
@@ -139,24 +140,6 @@ pub struct InitializeSubDaoV0<'info> {
   pub treasury_management_program: Program<'info, TreasuryManagement>,
   pub circuit_breaker_program: Program<'info, CircuitBreaker>,
   pub associated_token_program: Program<'info, AssociatedToken>,
-
-  /// CHECK: handled by thread_create
```
Comment (ThornM9) [inline programs/helium-sub-daos/src/instructions/initialize_sub_dao_v0.rs:143]: Is clockwork getting ripped out for now?

File: Anchor.toml (hunk 1 above)
Comment (ChewingGlass) [reply Anchor.toml]: Interesting. Reverted.

**Review by bryzettler — APPROVED** [review]

File: migration-docker/migrate.sh [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+anchor build -- --features no-genesis
+anchor deploy --provider.cluster $SOLANA_URL
+./init-idls.sh $SOLANA_URL
+
+pushd packages/helium-admin-cli && yarn link && popd
+
+TOTAL_STAKED_HNT=$(helium-admin sum-tokens --token staked_hnt --state export.json --decimals 8)
+TOTAL_UNSTAKED_HNT=$(helium-admin sum-tokens --token hnt --state export.json --decimals 8)
+TOTAL_HNT=$(echo "$TOTAL_STAKED_HNT + $TOTAL_UNSTAKED_HNT" | bc)
+TOTAL_MOBILE=$(helium-admin sum-tokens --token mobile --state export.json --decimals 8)
```
[the remark this answers is not in the document]
Comment (ChewingGlass) [reply migration-docker/migrate.sh:44]: Well, it's 8 decimals in the export since helium l1 has it as 8

**Comment by ChewingGlass** [comment]
Merging this, we will do a final review again pre-migration.

**Pull request merged**